.github/workflows/nightly.yml

```diff
@@ -1,57 +1,58 @@
 name: Nightly events
 
 # https://jasonet.co/posts/scheduled-actions/
+# https://github.community/t/distinct-job-for-each-schedule/17811/2
 on:
   schedule:
-    # At the end of every day
-    - cron: "0 0 * * *"
+    - cron: "0 0 * * 1-6" # 00:00 UTC MON-SAT
+    - cron: "0 0 * * 0" # 00:00 UTC SUN
```

Comment:
what is the difference of having Sun separately?

Reply:
Dockers will be rebuilt every sunday, `pypi release` will need to run everyday
so i split sunday separately to build dockers if it's on Sunday.